ReactWindows/ReactNative.Shared/Bridge/ReactInstance.cs

```diff
@@ -33,6 +33,9 @@ class ReactInstance : IReactInstance
 
         private bool _initialized;
 
+        public event TransitionToBridgeIdleCallback OnTransitionToBridgeIdleCallback;
```

Comment:
>public event [](start = 8, length = 12)

Do / should these events go in the `IReactInstance` interface? I don't know the answer to this.

Reply:
I was trying to track RN Android, and reconcile with a multi-Instance app architecture.

Reply:
Actually, looking at RN Android and another implementation -- yes. Otherwise, you'd end up casting the IReactInstance getter on the Context to the concrete. I'll fix this next.